devito/operator/profiling.py

```diff
@@ -364,13 +364,16 @@ def add_glb_fdlike(self, points, time):
         gpointss = gpoints/time
 
         if self.input:
+            traffic = sum(v.traffic for v in self.input.values())
             ops = sum(v.ops for v in self.input.values())
```

Comment:
nitpicking: blank line after this line? this would separate the known quantities (traffic, ops) from the derived metrics (gflops, gflopss, oi, ..)